dashboard/app/controllers/api/v1/pd/foorm_controller.rb

```diff
@@ -62,4 +62,15 @@ def get_submissions_as_csv
     csv = form.submissions_to_csv(submissions)
     send_csv_attachment(csv, filename)
   end
+
+  # POST api/v1/pd/foorm/validate_form
```

Comment:
Should this route be only accessible by a user that is a levelbuilder? It might be good to add some tests to make sure that if other types of users try to hit this route they can't

Reply:
good call